greet.js

```diff
@@ -0,0 +1,11 @@
+function greet(name) {
```

Comment:
You also haven't created a lab-jess directory for your work. You'll need to do that before I grade and merge your PR!